src/PropertyInfo/Tests/Data/HackData.hh

```diff
@@ -0,0 +1,433 @@
+<?hh
+
+/*
+ * (c) Mihai Stancu <stancu.t.mihai@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PropertyInfo\Tests\Data;
+
+/**
+ * Property Extractor.
```

Comment:
This class doesn't do anything it's just a dummy class used for the tests -- like `SetterDummy`. But I have updated the doc comment nonetheless.